lib/compilers/fpc.js

```diff
@@ -0,0 +1,25 @@
+var Compile = require('../base-compiler');
```

Comment:
Add copytight information to the top of the file. You can find it on every single file